README.md

```diff
@@ -17,6 +17,7 @@ We consume the product part from 4 different topics:
 
 ```json
 {
+  "id": "P1",
```

Comment:
Actually the "ProductDetails" does not need to have the id set. It will be overridden in code by the record's key.